src/SFA.DAS.ApprenticeCommitments.Api/Startup.cs

```diff
@@ -30,22 +26,27 @@ public class Startup
     {
         public Startup(IConfiguration configuration)
         {
+            Configuration = configuration;
+
             var config = new ConfigurationBuilder()
                 .AddConfiguration(configuration)
                 .SetBasePath(Directory.GetCurrentDirectory())
                 .AddEnvironmentVariables();
 
-            config.AddAzureTableStorage(options =>
+            if (!Configuration.IsAcceptanceTest())
```

Comment:
I'm not a massive fan of putting knowledge of tests in the main code.

Other ways of doing this might be to have a virtual method that adds all the "real life" code, and overide that in a `TestStartup.cs`.  

What do you think?

Reply:
this might prove to be more difficult because of where it is in code.  It's a lesser sin than the next issue

Reply:
Yes, I agree it's not as bad as the Sqlite dependency.  Happy to leave this as for future (!) improvement.